wpeverest/everest-forms
# Feature - Everest forms on top admin menu bar

### All Submissions:






### Changes proposed in this Pull Request:



Previously, We do not have the Everest forms menu on the admin top menu bar. This PR will introduce this feature.

### How to test the changes in this Pull Request:

1. switch to this branch and verify whether the Everest forms menu is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Everest forms on the top admin menu bar.
